labcodes/labstorybook
# DSYS-184: Add description to prop types for all components

**Link to task**
https://labcodes.atlassian.net/browse/DSYS-184

**Description**
This PR adds 222 prop type description for all 33 components or sub-components. In the description, component names are capitalized like `Search` (vs. the action/verb "search").

**How to test**
- With the storybook running, check each prop description for every component.
- Look for the pattern `/** */` on the Design System directory. You can use Visual Studio Code's search system or, from the terminal, run the following command:
```
 grep -rn "\/\*\*\ \*\/" labsystem/ 
 ```
 Then check if there are any descriptions missing.
 
 ---
 **Notes**:
 
 Currrently, there are some descriptions missing. The output of the `grep` command above should output:
 ```
 labsystem/src/Radio.js:24:    /** */
labsystem/src/Radio.js:26:    /** */
labsystem/src/Input/AbstractTextInput.js:45:    /** */
labsystem/src/Icon.js:66:    /** */
labsystem/src/Badge.js:49:    /** */
labsystem/src/Checkbox.js:27:    /** */
labsystem/src/Checkbox.js:29:    /** */
```
This is yet to be solved.  I leave some comments for further discussion:

- `AbstractTextInput.js`: shouldn't the prop `type` be required?
- `Badge/Item`: the prop `className` seem to be unused (it occours similiarly in `Checkbox` and `Toggle`).
- `Checkbox/Radio`: I couldn't describe the prop `value`. It seems to be unused and undocumented. (One more thing, there is a `Playground` inside a `Canvas` in the Stories)
- `Toggle`: shouldn't <input> have a `name` (like `Checkbox` and `Radio`)?
